assets/css/nav/_top_nav.scss

```diff
@@ -5,7 +5,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 0.75rem 1.5rem;
+  padding: 0;
```

Comment:
This seems like a big change? Does this squash the top-nav element, or was the padding getting overridden anyway?

Reply:
Oh before we merge I wanna check this real quick. So don't resolve this yet

Reply:
Okay this seems okay for now!

Reply:
I had been refreshing this page regularly, but still didn't see this request to _not_ merge at 4:50, which is when I merged it, because GH does not push comment updates. In the future, if you have an urgent request to not merge, you'll have to slack me!